### DISCUSSION TITLE
Where is the link for Beta 2.6.1 for Mac?

### DISCUSSION THREAD

**Discussion by fatrabbit-la** [artisan-roaster-scope/artisan#931] [General]
I read this version might resolve my issue with CHARGE and DROP flagging incorrectly but can only find download for 2.6.0.

**Comment by MAKOMO** [comment]
You find the potential unstable non-release developer version of the upcoming version always here: https://github.com/artisan-roaster-scope/artisan/releases/tag/continuous

**Reply by MAKOMO** [reply]
Note those builds are unsigned and not notarized. Thus on macOS you need to start them on the first time using a right-click and select "Open".

**Comment by fatrabbit-la** [comment]
Thank you. When do you anticipate this version being released as a complete build?

Fat Rabbit Coffee
***@***.*** ***@***.***>
www.fatrabbitcoffee.com <http://www.fatrabbitcoffee.com/>

**Reply by MAKOMO** [reply]
Not before September. But first I need to find a release sponsor...

**Comment by fatrabbit-la** [comment]
James Hoffman makes a mint off his various channels...

Fat Rabbit Coffee
***@***.*** ***@***.***>
www.fatrabbitcoffee.com <http://www.fatrabbitcoffee.com/>
